architecture/core_objects/builds.adoc

```diff
@@ -302,19 +310,19 @@ Copy the value returned from the above command and place it into the
 ----
 ====
 
-Then, create the `*Secret*` from the *_secret.json_* file using the following
+. Create the `*Secret*` from the *_secret.json_* file using the following
 command:
-
++
 ====
 ----
 $ osc create -f secret.json
 ----
 ====
 
-Add a `SourceSecret` field into the `Source` section inside the `BuildConfig`
+. Add a `SourceSecret` field into the `Source` section inside the `BuildConfig`
 and set it to the name of the `*Secret*` that you created, in this case
 `*scmsecret*`:
```

Comment:
Similar to the Docker procedure above, when and how was the name of the secret established?